api/routes/books.js

```diff
@@ -28,6 +32,14 @@ router.get('/:id', async (req, res, next) => {
 router.post('/', async (req, res, next) => {
   const status = 200
   try {
+    // const token = req.headers.authorization.split('Bearer ')[1]
+    //   const secret = new Buffer.alloc(SECRET_KEY, "base64")
+    // const payload = jwt.verify(token, SECRET_KEY)
+    // const user = await Users.findOne({ _id: payload.id }).select('-__v -password')
+    // const isAdmin = (user.admin === true)
+    // const adminStatus = 404
+    // if(!isAdmin) throw new Error(`You do not have access ${adminStatus}`)
+
```

Comment:
What was happening here that wasn't working? My guess would be the `user.admin === true` bit...?